test/AutoDiff/superset_adjoint.swift

```diff
@@ -33,6 +33,14 @@ SupersetVJPTests.test("CrossModuleClosure") {
   expectEqual(1, gradient(at: Float(1)) { x in x + 2 })
 }
 
+SupersetVJPTests.test("SubsetOfSubset") {
+  @differentiable(wrt: (x, z))
+  func foo(_ x: Float, _ y: Float, _ z: Float) -> Float {
+    Float(0).withoutDerivative()
```

Comment:
`Differentiable.withoutDerivative()` has now been removed. Shouldn't this be `withoutDerivative(at: Float(0))` instead?

Reply:
Good catch!